<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>controller</class>
 <widget class="QMainWindow" name="controller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>controller</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>数据日志</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Bsearch">
            <property name="maximumSize">
             <size>
              <width>85</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BclearSearch">
            <property name="text">
             <string>取消搜索</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LEsearch"/>
          </item>
          <item>
           <widget class="QPushButton" name="BcleanDialog">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>清除数据日志</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/QIcon/clean.png</normaloff>:/QIcon/clean.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="Edialog">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>数据发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BcleanSend">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>清除发送数据</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/QIcon/clean.png</normaloff>:/QIcon/clean.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QTextEdit" name="EtextSend"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="Bsend">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>发送</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BsendByDialog">
              <property name="text">
               <string>按日志发送</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BsendOnTime">
              <property name="text">
               <string>定时发送</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BsendFile">
              <property name="text">
               <string>发送文件</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QGroupBox" name="GBnetSet">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>网络设置</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="LnetSetTypePro">
             <property name="text">
              <string>协议类型</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="CBtypePro">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
             <property name="currentText">
              <string>TCP</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <property name="duplicatesEnabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>TCP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UDP</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lip">
             <property name="text">
              <string>本地主机地址</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="LElocalHostAdd">
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>127.0.0.1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="BnetSetCleanHostIP">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/QIcon/clean.png</normaloff>:/QIcon/clean.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="BnetSetPasteHostIP">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/QIcon/copy.png</normaloff>:/QIcon/copy.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="Lport">
             <property name="text">
              <string>本机主机接口</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LElocalHostPort">
             <property name="text">
              <string>8899</string>
             </property>
             <property name="cursorPosition">
              <number>4</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LEcastSelect">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>客户端列表</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="CBclientList">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>未有客户端连接</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="GBreceiveSet">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>接收设置</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>150</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="title">
              <string>显示设置</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QRadioButton" name="RBreceiveSetAsc">
                <property name="text">
                 <string>ASCⅡ</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="RBreceiveSetHex">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>HEX</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="CBhideDialog">
               <property name="text">
                <string>接收数据不显示</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QCheckBox" name="CBrecepSetSaveDialog">
                 <property name="text">
                  <string>日志保存至文件</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="BrecepSetSaveDialog">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>选择保存至</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>发送设置</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QGroupBox" name="GBsendSetInputType">
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="title">
              <string>显示设置</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QRadioButton" name="RBsendSelASC">
                <property name="text">
                 <string>ASCⅡ</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="RBsendSelHEX">
                <property name="text">
                 <string>HEX</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="title">
              <string>发送端选择</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QRadioButton" name="RBsendZoneClient">
                <property name="text">
                 <string>客户端</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="RBsendZoneServer">
                <property name="text">
                 <string>服务器端</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QCheckBox" name="CBsendSetrepPeriod">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>数据流循环发送</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="LEsendSetrepPeriod">
               <property name="clearButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>毫秒</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="CBsendSetclearAftSend">
             <property name="text">
              <string>发送完成自动清空</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="CBsendAddHead">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>发送时添加信息头</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QPushButton" name="BactivateListen">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>启动监听</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="BserverInitial">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>服务器重置</string>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="BsendSetConnect">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>连接</string>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="BsendSetBreak">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>断开</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
          <zorder>GBsendSetInputType</zorder>
          <zorder>CBsendSetclearAftSend</zorder>
          <zorder>groupBox_4</zorder>
          <zorder>CBsendAddHead</zorder>
          <zorder>widget</zorder>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
